shell/app/electron_main_win.cc

```diff
@@ -2,12 +2,11 @@
 // Use of this source code is governed by the MIT license that can be
 // found in the LICENSE file.
 
-#include <windows.h>  // windows.h must be included first
-
 #include <atlbase.h>  // ensures that ATL statics like `_AtlWinModule` are initialized (it's an issue in static debug build)
 #include <shellapi.h>
 #include <shellscalingapi.h>
 #include <tchar.h>
+#include <windows.h>  // windows.h must be included first
```

Comment:
literally the comment!!

Reply:
i'm aware - that's why this PR is in a draft state. These changes were applied automatically and as can be seen in commits, i'm going through and progressively addressing these issues as they crop up as well as determining how best to align to Chromium.